src/components/side_panel/data_validation/dv_criterion_form/dv_input/dv_input.xml

```diff
@@ -1,20 +1,25 @@
 <templates>
   <t t-name="o-spreadsheet-DataValidationInput">
-    <div class="o-dv-input position-relative w-100">
-      <input
-        type="text"
-        t-ref="input"
-        t-on-input="onValueChanged"
-        t-att-value="props.value"
-        class="o-input"
-        t-att-class="{
-            'o-invalid border-danger position-relative': errorMessage,
-          }"
-        t-att-title="errorMessage"
-        t-att-placeholder="placeholder"
-        t-on-keydown="props.onKeyDown"
-        t-on-blur="props.onBlur"
-      />
+    <div class="o-dv-input position-relative w-100 p-1">
+      <t t-if="props.criterionType !== 'isValueInList'">
```

Comment:
nitpick: probably testing if the registry has `allowedValues === "onlyLiterals"` is more robust